devicon.json

```
@@ -3297,6 +3297,32 @@
         "color": "#5E5185",
         "aliases": []
     },
+    {
+        "name": "hadoop",
+        "altnames": [
+            "Apache Hadoop"
+        ],
+        "tags": [
+            "framework", 
+            "big data"
+        ],
+        "versions": {
+            "svg": [
+                "original", 
+                "plain"
+            ],
+            "font": [
+                "plain"
+            ]
+        },
+        "color": "#ffff00",
+        "aliases": [
+            {
+                "base": "plain",
+                "alias": "line"
+            }
+        ]
```

```suggestion
        "aliases": []
```
I think `plain` version can't be aliased as `line` version. So I suggest this change.

---

Why can't it? :O

---

hmm, I don't agrre :/ If anything I think it should qualify as a line version, and possibly not plain version. 
The reason being that the plain version is usually just the original version, but mono-colored and with unified paths. Meanwhile the line version is (in my interpretation) just the inverse of the plain icon. So where there are empty spots in the plain version, there will be lines in the line version, and vice-versa.

I'm curious what others think about this though. 
@Panquesito7 @kilianpaquier Any thoughts on whether or not [this icon](https://github.com/devicons/devicon/pull/1488/files#diff-613b72c291b19bc5cae774fa507bd1c71713db1822749fb5ff8314eb54553ee9) should qualify for plain and/or line versions?

---

Hello @Snailedlt, I don't see why it couldn't be aliases as "line" icon. There's only one path, and has only lines ?

---

Thanks for your input. I agree with this